src/array.h

```diff
@@ -2,11 +2,14 @@
 #include <cstddef>
 #include <cstring>
 #if __cplusplus > 201703L
-#   include <array>
-namespace bn256 {
-template <typename T, std::size_t S>
-using array = std::array<T, S>;
-}
+   #include <span>
+   #include <cstdint>
+   #include <tuple>
```

Comment:
Why not just always include `<tuple>` and `<cstdint>`

Reply:
Well, it was not needed in the other path (where `<bn256/span.h>` is included), and I preferred not to make unneeded changes. Hopefully, after we do switch to build with c++20 enabled, we can remove the `#if` and the alternate path so `<tuple>` and `<cstdint>` will always be included